ynput/ayon-core
# Chore: Move deadline client code

## Changelog Description
Moved Deadline client code next to server codebase.

## Testing notes:
1. Create package of Deadline and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Deadline integration should work as before.
